islandora_openseadragon.install

```diff
@@ -38,3 +54,21 @@ function islandora_openseadragon_uninstall() {
   );
   array_walk($variables, 'variable_del');
 }
+
+/**
+ * Implements hook_update_N().
+ *
+ * Add the required table for handling authentication tokens.
+ * This is the first instance that has this table.
+ */
+function islandora_openseadragon_update_7001(array &$sandbox) {
+  $t = get_t();
+  // Variable structure has changed.
+  $variables = array(
+    'islandora_openseadragon_settings',
+    'islandora_openseadragon_tile_size',
+    'islandora_openseadragon_tile_overlap',
+  );
+  array_walk($variables, 'variable_del');
```

Comment:
Instead of just deleting, should this be attempting to crosswalk values, and likely explicitly select djatoka?